website/server/controllers/api-v3/members.js

```diff
@@ -270,6 +270,45 @@ api.getChallengeMemberProgress = {
 };
 
 /**
+ * @api {get} /api/v3/members/:toUserId/objections-to/:interaction Get the message of any errors that would occur if the given interaction was attempted
```

Comment:
I'm not sure there is anything to add. The example is of a method with a much more complicated parameter structure; the current documentation for the new method already includes everything about its parameters. The only missing information is the exception, and I don't think committing to a specific exception here is a good idea.